docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: '3.7'
+
+services:
+  reviewer-reporter:
+    build:
+      context: .
+    image: liberoadmin/reviewer-reporter:latest
```

Comment:
this will change to a commit-based tag when running in CI

Reply:
Ah yes - but I'm hoping the github actions will manage the push to docker and also call it latest.